### ISSUE TITLE
Non wasi?

### ISSUE THREAD

**Issue by elix1er** [RustPython/RustPython#5380]
## Summary

wasm32-unknown-unknown no_std non-wasi target possible?

**Comment by youknowone** [comment]
non-wasi wasm32 target is supported. no_std is not supported in any platform
